qa/tasks/cephfs/test_forward_scrub.py

```diff
@@ -46,6 +47,12 @@ def _get_paths_to_ino(self):
 
         return inos
 
+    def _check_for_presence_of_MDS_damage(self):
+        return self.assertIn("MDS_DAMAGE", self.mds_cluster.mon_manager.get_mon_health()['checks'])
```

Comment:
return an assert?

This should be

```
return "MDS_DAMAGE" in self.mds_cluster.mon_manager.get_mon_health()['checks']
```

Reply:
I thought return value should be a boolean and assert also returns a boolean. that's why used it.